### DISCUSSION TITLE
Kazanç

### DISCUSSION THREAD

**Discussion by Sehoo72** [IBAX-io/go-ibax#1316] [Soru Cevap - Türkçe]
Peki Herkes Bu İbax Airdropu kazanacakmı yoksa ilk, 1000kisimi

**Comment by Skips1** [comment]
Kanka buda bi soru Açıklamalarda  en çok kişi getirenler ve çekiliş olacak yazıyor burdaki adminlerde puanına göre herkes alıcak diyor 1 şunatta göreveğiz artık kanka.

**Comment by Fthhaydnn** [comment]
İlk açtığım cuzdanla guncellemeden sonraki cüzdan adresi farklı oldu. Yeni adres açmama rağmen. Ayrıca şifre aynı şifre yeni şifre olusturmadim. Nasıl oldu anlamadim yardımcı olacak var mi
